src/containers/login_page/LoginPage.js

```diff
@@ -123,7 +123,7 @@ function LoginPage({history}) {
             id='password' type='password' value={password} 
             onChange={(e) => passwordHandler(e)} 
             placeholder="Пароль" required/>        
-        <button type="submit" onClick={onSubmit} disabled={formValid}>
+        <button className='blue-button' type="submit" onClick={onSubmit} disabled={formValid}>
```

Comment:
Try to name classnames the way everyone could understand what is the purpose of that element. In your case 'blue-button' means nothing :)) It's better to name it 'submit-button' or smth like that :)